thingooConnector/thingooconnector.py

```diff
@@ -0,0 +1,42 @@
+from thingooConnector.connector import Connector
+
+from thingooConnector.httpconnector import HTTPConnector
+from thingooConnector.mqttconnector import MQTTConnector
+
+
+class ThingooConnector(Connector):
+    def __init__(self, host, device_info, entities, client_credentials, userdata=None, http_only=False):
+        """
+        :param host: Thingoo host
+        :param device_info: A device info object
+        :type device_info: :class:`DeviceInfo`
+        :param entities: A list of entities
+        :param client_credentials: A ClientCredentials object used to get access_token
+        :type device_info: :class:`ClientCredentials`
+        :param userdata: A UserData object used to connect to MQTT
+        :type device_info: :class:`UserData`
+        :param http_only: A boolean value that indicates if the device should only use http without connecting to mqtt
+        :type http_only: bool
+        """
+        if userdata is None:
```

Comment:
I would rename userdata to mqtt_credentials for example

Reply:
Resolved in aabc133